mdflynn/rancid-tomatillos
# Feature/create form

## Is this a fix or a feature?
- Fix/Feature

## What is the change?
- A form was created to search for movies
- The movies rendered to the page will only include movies that match searched results

## What does it fix?
- Cleans up some styling issues

## Where should the reviewer start?
- App.js: Lines 16, 23-24, 28-31, 40-61, 66-70
- Form.js: Lines 1-45,
- Form.scss: Lines 1-66
- MovieCard.scss: Lines  1, 11-17
- Nav.js: Lines 10-11
- Nav.scss: Lines 6, 11, 17-19, 24-26, 31-37

## How should this be tested?
- Clone down the repo
- `cd` into the directory
- Run `npm start` to spin up a server
- Navigate over to the localhost provided 
- Type into the form on the top of the page and observe how the movies are filtered based on which titles are typed in

## Issues Resolved
- #5
